src/main/java/com/iexec/worker/docker/DockerRegistryConfiguration.java

```diff
@@ -31,12 +32,23 @@
 @Slf4j
 @Configuration
 @ConfigurationProperties(prefix = "docker")
+@Getter
+@Setter
 public class DockerRegistryConfiguration {
 
-    @Setter
-    @Getter
     private List<RegistryCredentials> registries;
 
+    /**
+     * Min pull timeout expressed in minutes
+     */
+    @Value("${docker.image.pull-timeout.min:5}")
+    private long minPullTimeout;
+    /**
+     * Max pull timeout expressed in minutes
+     */
+    @Value("${docker.image.pull-timeout.max:30}")
```

Comment:
I might be missing something, do we need a default value here?
```suggestion
    @NonZero
    @Value("${docker.image.pull-timeout.max")
```
(max should be > than min also but we can drop it if you want)

Reply:
We don't really need it. Let it be defined in `application.yml`.
Regarding validation, see https://github.com/iExecBlockchainComputing/iexec-worker/pull/442/commits/22f869f54a7243640cf874ec02a16f2838f8c66d.